### DISCUSSION TITLE
TypeError: Cannot read properties of null (reading 'insertBefore')

### DISCUSSION THREAD

**Discussion by the-lgtm** [community/community#129434] [Programming Help]
### Body

I am trying to render a list of items in my NuxtJS application.

```vue
<Product v-for="product in products.items" :product="product" />
```

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by yhemn** [comment]
You need to add a key attribute in your product component

```vue
<Product v-for="product, index in products.items" :product="product" :key="index" />
```

**Comment by ebndev** [comment]
Hi @the-lgtm, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
